cms/datavis/blocks/annotations.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any
+
+from wagtail import blocks
+from wagtail.blocks.struct_block import StructValue
+
+from cms.datavis.blocks.utils import TextInputFloatBlock, TextInputIntegerBlock
+
+
+class PointAnnotationStructValue(StructValue):
+    def get_config(self) -> dict[str, Any]:
+        # The SVG coordinate system measures from the top left. It makes more
+        # sense to users to measure from the bottom left, to match the chart
+        # coordinates.
+        if label_offset_y := self.get("label_offset_y"):
+            label_offset_y = label_offset_y * -1
```

Comment:
Non-blocking nit-pick, would this read better?
```suggestion
            label_offset_y = -label_offset_y
```

Reply:
That's a good suggestion, thanks. I have changed this in 46cd3ef.